backend/api/Services/MissionSchedulingService.cs

```diff
@@ -452,32 +455,8 @@ private async Task StartMissionRun(MissionRun queuedMissionRun)
             return ongoingMissions;
         }
 
-        private async Task<bool> TheSystemIsAvailableToRunAMission(string robotId, string missionRunId)
+        private bool TheSystemIsAvailableToRunAMission(Robot robot, MissionRun missionRun)
         {
-            bool ongoingMission = await OngoingMission(robotId);
```

Comment:
I think we should still use robotId, but if we do not care if there is an ongoing mission (that is waiting for the updates) we should not check it here